phovea_server/util.py

```diff
@@ -46,6 +46,12 @@ def to_json(obj, *args, **kwargs):
   if 'indent' in kwargs:
     del kwargs['indent']
   kwargs['ensure_ascii'] = False
+
+  if 'use_decimal' in kwargs:
```

Comment:
`use_decimal` is not a good name

Reply:
which one would be a better fit? `encode_decimal`?

Reply:
or `custom_encoder`?